website/docs/general/sidebars.json

```
@@ -1,16 +1,27 @@
 {
   "docs": {
-    "General": [
-      "community",
+    "Community": [
+      "join",
       "events",
-      "contributor-guide",
-      "committer-guide",
-      "subscribe-guide",
-      "blog",
-      "security",
-      "2fa",
-      "release-guide",
-      "integrate-with-project-docs"
+      {
+        "type": "category",
+        "label": "Contribute to APISIX",
+        "items": [
+          "submit-issue",
+          "contributor-guide",
+          "blog",
+          "security",
+          "committer-guide"
+        ]
+      },
+      {
+        "type": "category",
+        "label": "Process and style guides",
+        "items": [
+          "documentation-guide",
```

If you decide to update https://github.com/apache/apisix-website/pull/974/files#r836146718, please update it here as well.
```suggestion
          "documentation-style-guide",
```